cpp/src/io/json/json_column.cu

```diff
@@ -430,11 +465,12 @@ void make_device_json_column(device_span<SymbolT const> input,
                              bool is_array_of_arrays,
                              bool is_enabled_lines,
                              bool is_enabled_mixed_types_as_string,
+                             cudf::io::json_reader_options const& options,
```

Comment:
suggestion: I think we should update the function description to include this parameter

Reply:
updated.